nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -58,6 +58,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </ItemGroup>
   </Target>
 
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.Windows.CsWinRT.Embedded.targets" />
```

Comment:
The condition was removed/this import was moved up in an experiment for P2 scenarios. Unintentionally merged those experiments into this branch. This can get the condition and go back to the others. We needed it here because it exposed properties I was using to affect how the cswinrt tool handled embedded scenarios.